ref-cache.md

```diff
@@ -2,3 +2,50 @@
 permalink: /ref-cache.html
 ---
 
+# Loc-I Reference Integrated Cache
+
+## What
+
+Central reference cache that loads a set of reference spatial layers directed by the Loc-I steering committee.
```

Comment:
Central cache holding a set of spatial layers to serve as a reference for other applications. The layers included are as directed by the Loc-I steering committee.

Reply:
added that in.